vnext/Microsoft.ReactNative.SharedManaged/JSValueReaderGenerator.cs

```diff
@@ -232,41 +255,50 @@ private static Delegate GenerateReadValueForClass(Type valueType)
       //       }
       //     }
       //   }
+      //
+      //   return value;
       // }
 
-      var valueTypeInfo = valueType.GetTypeInfo();
-      bool isStruct = valueTypeInfo.IsValueType && !valueTypeInfo.IsEnum;
-      bool isClass = valueTypeInfo.IsClass;
-      if (isStruct || (isClass && valueType.GetConstructor(Type.EmptyTypes) != null))
-      {
-        var fields =
-          from field in valueType.GetFields(BindingFlags.Public | BindingFlags.Instance)
-          where !field.IsInitOnly
-          select new { field.Name, Type = field.FieldType };
-        var properties =
-          from property in valueType.GetProperties(BindingFlags.Public | BindingFlags.Instance)
-          let propertySetter = property.SetMethod
-          where propertySetter.IsPublic
-          select new { property.Name, Type = property.PropertyType };
-        var members = fields.Concat(properties).ToArray();
-
-        return ReadValueDelegateOf(valueType).CompileLambda(
-          Parameter(typeof(IJSValueReader), out var reader),
-          Parameter(valueType.MakeByRefType(), out var value),
-          value.Assign(New(valueType)),
-          IfThen(Equal(reader.Property(ValueType), Constant(JSValueType.Object)),
-            ifTrue: AutoBlock(
-              Variable(typeof(string), out var propertyName),
-              While(reader.Call(GetNextObjectProperty, propertyName),
-                (members.Length != 0)
-                ? Switch(propertyName,
-                    members.Select(member => SwitchCase(
-                      value.SetPropertyStatement(member.Name, reader.CallExt(ReadValueOf(member.Type))),
-                      Constant(member.Name))).ToArray()) as Expression
-                : Default(typeof(void))))));
-      }
+      var fields =
+              from field in classType.GetFields(BindingFlags.Public | BindingFlags.Instance)
```

Comment:
nit: seems indention seems to has changed
```suggestion
        from field in classType.GetFields(BindingFlags.Public | BindingFlags.Instance)
```

Reply:
Good catch! Fixed.

---
In reply to: [414053528](https://github.com/microsoft/react-native-windows/pull/4696#discussion_r414053528) [](ancestors = 414053528)